fastlane/Pluginfile

```diff
@@ -2,7 +2,7 @@
 #
 # Ensure this file is checked in to source control!
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.9.14'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: 'ios-l10n-lint'
```

Comment:
⚠️ This needs to be reverted to point to a tag once wordpress-mobile/release-toolkit#177 is merged and we tag a new version of the release toolkit, before we can merge this PR.